### ISSUE TITLE
Audit a11y requirements

### ISSUE THREAD

**Issue by lmorchard** [mozilla/FirefoxColor#47]
From [acceptance criteria](https://github.com/mozilla/ThemesRFun/blob/master/docs/acceptance.md):

> - [ ] All buttons and links should have visible focus states
> - [ ] All buttons and links should be accessible via keyed entry (tab selection)
> - [ ] All form elements should include appropriate label attributes
> - [ ] All grouped buttons should be nested in a <fieldset> and described with a legend
> - [ ] All UI should be verified to use A11y friendly contrast ratios

Filing this issue as a checklist item for near the end, though it should be an aspect we mind all along the way

**Comment by johngruen** [comment]
I'd also recommend running this through chrome's new a11y auditor

**Comment by pdehaan** [comment]
Another one I really like is **sonarwhal**, although I assume all of these tools are using [aXe](https://www.deque.com/products/axe/) for a11y linting.
Here's the results for <https://mozilla.github.io/ThemesRFun/>:

https://sonarwhal.com/scanner/1d9be01e-d557-46f6-a5b7-7f2d3a5f85a5

![sonarwhal-axe](https://user-images.githubusercontent.com/557895/36608167-d8b0c63a-187d-11e8-953c-e22b06820f2b.png)

---

Also, https://github.com/GoogleChrome/lighthouse (which you can use in Chrome DevTools, as a Chrome extension, or using the Node CLI):

https://googlechrome.github.io/lighthouse/viewer/?gist=9e8c100be008e6497da32952be3a9522

![lighthousereport](https://user-images.githubusercontent.com/557895/36608526-f42ef4b2-187e-11e8-832e-b823ab7ce211.png)
